test/os/ui/draw/draw.test.js

```diff
@@ -0,0 +1,41 @@
+goog.require('ol.Feature');
+goog.require('ol.geom.Polygon');
+goog.require('ol.proj');
+goog.require('os.map');
+goog.require('os.ui.draw');
+
+describe('os.ui.draw', function() {
+  window.localStorage.clear();
+  os.net.RequestHandlerFactory.addHandler(os.net.SameDomainHandler);
+
+  beforeEach(function() {
+    window.localStorage.clear();
+  });
+
+  it('should read grid settings as numbers', function() {
+    runs(function() {
+      var gridDetail = os.ui.draw.getGridSetting(os.ui.draw.GRID_DETAIL, 0.1);
+      expect(gridDetail).toBeDefined();
+      expect(typeof gridDetail == 'number').toBeTruthy();
```

Comment:
While this works, if this were to fail, the log would contain "Expected 'false' to be 'true'". However, if you write this as `expect(typeof gridDetail).toBe('number')` then the log would be something like "Expected 'string' to be 'number'", which is closer to the test that is actually happening here.

Generally, if you find yourself putting `==` or `===` in the expectation, you probably mean to use `expect(a).toBe(b)` (for non-strict equality or deep equality checks, use `expect(a).toEqual(b)`).

Reply:
Rewrote to no longer pull an actual setting; now it uses a true .toBe as recommended.